index.html

```diff
@@ -23,12 +23,15 @@
           <ul class="nav-menu">
             <li class="nav-item">
               <a class="link" href="#portfolio">Portfolio</a>
+              <img class="seperator" src="/images/List Item/Width Bottom.png" alt="" srcset="">
             </li>
             <li class="nav-item">
               <a class="link" href="#about">About</a>
+              <img class="seperator" src="/images/List Item/Width Bottom.png" alt="" srcset="">
             </li>
             <li class="nav-item">
               <a class="link" href="#contact">Contact</a>
+              <img class="seperator" src="/images/List Item/Width Bottom.png" alt="" srcset="">
             </li>
             <li class="nav-item last-menu-item">
               <a class="link" href="#msg">
```

Comment:
- [ ] As stated before by the previous reviewer, kindly make sure to add a separator like in the Figma design.

- [ ] [OPTIONAL] I suggest using the correct font type and weight like the Figma design.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/210346379-c771225c-cee6-4843-a7db-b7557a673870.png)  | ![image](https://user-images.githubusercontent.com/53402600/210346304-5fa8c131-dbae-460c-b1a4-4f726a7c61c0.png) |